Promact.Oauth.Server/src/Promact.Oauth.Server/Models/AppSettings.cs

```diff
@@ -10,12 +10,10 @@ public class AppSettings
 
         public string Host { get; set; }
 
-        public int Port { get; set; }
+        public string Port { get; set; }
```

Comment:
I have update **EmailCrednetials** class, Now **Port** as number and **SslOnConnect** as boolean